doc/migration.md

```diff
@@ -0,0 +1,107 @@
+   2. Spot check a few records and make sure the metadata is correct, especially those that reference an external or local authority (e.g., creators, subjects, genres, etc.). 
+   3. Check the ADRL EZID account (sb-adrl) to confirm that no additional ARKs have been created. 
+
+7. Ingest the collection records for harvest-based collections from adrl-dm master branch.
+
+8. Verify that the collection records were successfully ingested.
+   1. Spot check the records to make sure the metadata is correct.
+   2. Check the ADRL EZID account (sb-adrl) to confirm that no additional ARKs have been created. 
+
+9. Ingest harvest-based collections. (If new ETDs are in the queue, they may also be ingested post-migration.)
+
+10. Verify that each harvest collection has been successfully ingested. Do not move on to the next collection until you are sure that the records are correct. 
+    1. Compare collection object counts to what was documented at the beginning of the process. If there was a discrepancy between ADRL and Alma counts at the beginning of the process (because of queued content), confirm against the Alma count. 
+    2. Spot check a few records and make sure the metadata is correct, especially those that reference an external or local authority (e.g., creators, subjects, genres, etc.). 
+    3. Check the ADRL EZID account (sb-adrl) to confirm that no additional ARKs have been created. 
+   
+11. (Step may also be completed post-migration.) Ingest any new CSV, MODS, or ETDs objects collection-by-collection. Repeat this step, and the next, until all new CSV and MODS objects are ingested.
+    1. If adding records to an existing collection, it is not necessary to re-ingest the collection record. 
+    2. If adding a new collection, ingest the collection record first, followed by the object records. (Note that scanned map collection records are stored in a separate directory from the scanned map object records.) 
+
+12. (Step may also be completed post-migration.) Following the ingest of each new batch of CSV or MODS objects, verify that it has been successfully ingested. Do not move on to the next batch until you are sure that the records are correct. 
+    1. Compare by collection object count to what was documented at the beginning of the process. 
+    2. Spot check a few records and make sure the metadata is correct, especially those that reference an external or local authority (e.g., creators, subjects, genres, etc.). 
+    3. Check the ADRL EZID account (sb-adrl) to confirm that the correct number of new ARKs have been minted for the objects and collection records ingested.
+
+## Post Migration 
+
+1. Export brief object records from new production environment (produces CSV file with following fields: type, id, accession_number, identifier, title, access_policy). Check that no demo arks exist in the new production system (demo arks begin with ark:/99999)
+Run: `bin/export-identifiers`
```

Comment:
@dunn: can you confirm that this is correct? noticed it wasn't the same command issued above (but didn't know if that was because it was being issued in the new production system vs. the old production system)

Reply:
It will also be `RAILS_ENV=production bin/export/export-identifiers`